### DISCUSSION TITLE
New version of Mojoportal

### DISCUSSION THREAD

**Discussion by SinaBina** [i7MEDIA/mojoportal#150] [General]
Hello
As an old supporter of Mojoportal from its very beginning I have always had eyes on its development.I know that a lot is happening behind the shelf but can you please elaborate more on any news regarding new versions and the future of Mojoporal? There has been no release for past 5 years. Should we continue investing on Mojoportal?

Thank you
Regards

**Comment by JosephMDavis** [comment]
Hi Sina,

We're working on a release. As you probably noticed, the repo isn't dead, we've been actively enhancing mojo. Where we've fallen flat is publishing releases.

The reason for our lack of published releases is we are still supporting 4 database platforms with no ORM. This increases all development which touches a data layer by 4x as much time.

We've talked about switching mojo to Entity Framework or something like it but we haven't done it because of the amount of time it will take to convert a project of this size. 

So, I have a couple of people working their way through a beta release and logging bugs. You can see that in the our [mojoPortal 2.9 project](https://github.com/orgs/i7MEDIA/projects/2). I'm triaging as they go to push enhancements and bugs of lesser importance to our [mojoPortal 2.9.1 project](https://github.com/orgs/i7MEDIA/projects/5).

If you're inclined to help, we'd welcome it with open arms.

Thank you,
Joe
